{
  Failure failure=new Failure("index","type","id",new RuntimeException("test"));
  String toString=failure.toString();
  assertThat(toString,containsString("\"type\":\"runtime_exception\""));
  assertThat(toString,containsString("\"reason\":\"test\""));
  assertThat(toString,containsString("\"status\":500"));
}
